src/shared/codelens/localLambdaRunner.ts

```diff
@@ -449,11 +460,11 @@ export async function invokeLambdaFunction(
     }
 }
 
-const getConfig = async (params: {
+export async function getConfig(params: {
```

Comment:
I'm a little conflicted if this should be exported from this file; I'm thinking we should move this to a new file. The reasoning being, this config file was originally JUST for run/debug concerns, but now this has expanded its scope. Consider a `samUtils.ts` file?

Reply:
Yes, and suggest to rename it to something more specific, like "getSamLocalDebugConfiguration"

Reply:
Also move any functions that consume from this config to that file as well and export them.